Classes/Form/Field/Text.php

```diff
@@ -39,6 +40,11 @@ class Text extends Input implements FieldInterface
     /**
      * @var string
      */
+    protected $richtextConfiguration;
```

Comment:
This property could have the constant as default value, this would make `getRichtextConfiguration()` a bit smaller and faster for default usages.